tests/cpp/gen_4in2out.py

```diff
@@ -0,0 +1,46 @@
+from os import path, mkdir
+import numpy as np
+import sys
+
+import nnvm.symbol as symbol
+import nnvm.graph as graph
+import nnvm.compiler
+from nnvm.frontend import from_mxnet
+
+import tvm
+
+data1 = symbol.Variable(name="data1")
```

Comment:
Why do we need this file? It looks to me that they just generate the artifact. We probably should generate them only when we are doing the tests.